pkg/volume/volume.go

```diff
@@ -30,6 +30,15 @@ type Volume interface {
 	GetPath() string
 }
 
+// ReadWriteMode represents the usage mode for a volume.
+type ReadWriteMode string
```

Comment:
Do we want to name this `FsPermissions` instead? This type is conveying more than just `read` or `read-write` modes. For example `managed` mode.

How about a struct with `permissions` and `management` mode?